frontend/src/metabase/search/components/SearchResult.jsx

```diff
@@ -107,12 +107,12 @@ function Score({ score }) {
   return (
     <pre className="hide search-score">
       {`\n\n
```

Comment:
My one concern is that a vector of scores is quite easy to mix up. Is it possible for this to be a map/object so that the scores are less ambiguous than `[1 3 4.3 ...]` `{:text 1 :pinned 3 ...}` would be far better i'm thinking. I know things are scored using a vector but it might be a bit nicer to keep this as a map/object almost everywhere (with a schema type on it) and just a single helper function that knows how to turn it into a vector and only in particular places where the vector is scored.

Reply:
Fair point. I don't want to lose too much sleep over it since it's 100% for debugging, but I just pushed up a new commit that makes the display a lot more robust:

![image](https://user-images.githubusercontent.com/784417/109538071-46ae5000-7a85-11eb-98b2-5a0c204ccb37.png)